### PR TITLE
Add new features

### PR DESCRIPTION
**Pull request by Mahua233**

### CODE REVIEW

**Comment by frankhjwx** [comment]
Rework the class names and this branch can be merged

File: Storyboard Engine/StoryboardObject.py [hunk 1]
```diff
@@ -142,12 +146,54 @@ def startLoop(self, start_t, loop_count):
...
 
+
     def printObj(self):
-        self.codes.insert(0, ','.join(
-            [self.type, self.placement, self.alignment, self.file_name, str(self.x), str(self.y)]))
+        self.codes.insert(0, ','.join([self.type, self.placement, self.alignment, self.file_name, str(self.x), str(self.y)]))
         for code in self.codes:
             print(code)
 
+class eventObj():
```
Comment (frankhjwx) [inline Storyboard Engine/StoryboardObject.py]: Change the class name to **Code**, despite of getList func, also add a function called **getString()**, which can return a string of the Code

File: Storyboard Engine/StoryboardObject.py [hunk 2]
```diff
@@ -142,12 +146,54 @@ def startLoop(self, start_t, loop_count):
...
+            self.timing = list(timing)
+        else:
+            self.timing = Timing(timing)
+        self.data = data
+
+    def getList(self):
+        # return a list look like [key, easing, start_t, end_t, *data]
+        return flatten([self.key, self.easing, self.timing, self.data])
+
+class ObjectManage():
```
Comment (frankhjwx) [inline Storyboard Engine/StoryboardObject.py]: change the Class name to **Scene**

File: Storyboard Engine/StoryboardObject.py (hunk 1 above)
Comment (Mahua233) [reply Storyboard Engine/StoryboardObject.py]: The getList func return a list of object, so idk override __str__ func maybe useful.
Class name will change. Wait me next commit.

File: Storyboard Engine/StoryboardObject.py (hunk 2 above)
Comment (Mahua233) [reply Storyboard Engine/StoryboardObject.py]: ok

**Review by frankhjwx — APPROVED** [review]
hao

**Pull request merged**